https://bugs.gentoo.org/591392
https://bugzilla.mindrot.org/show_bug.cgi?id=2590

7.3 added seccomp support to MIPS, but failed to handled the N32
case.  This patch is temporary until upstream fixes.

--- openssh-7.3p1/configure.ac
+++ openssh-7.3p1/configure.ac
@@ -816,10 +816,10 @@ main() { if (NSVersionOfRunTimeLibrary("
 		seccomp_audit_arch=AUDIT_ARCH_MIPSEL
 		;;
 	mips64-*)
-		seccomp_audit_arch=AUDIT_ARCH_MIPS64
+		seccomp_audit_arch=AUDIT_ARCH_MIPS64N32
 		;;
 	mips64el-*)
-		seccomp_audit_arch=AUDIT_ARCH_MIPSEL64
+		seccomp_audit_arch=AUDIT_ARCH_MIPSEL64N32
 		;;
 	esac
 	if test "x$seccomp_audit_arch" != "x" ; then
